app/service/ingest.py

```diff
@@ -167,15 +167,24 @@ def validate_entity_relationships(data: dict[str, Any]) -> None:
         for fam in data["families"]:
             families.append(fam["import_id"])
 
-    documents = []
+    document_family_import_ids = []
     if "documents" in data:
-        for doc in data["documents"]:
-            documents.append(doc["family_import_id"])
-
-    family_document_set = set(families)
-    unmatched = [x for x in documents if x not in family_document_set]
-    if unmatched:
-        raise ValidationError(f"No family with id {unmatched} found")
+        for entity in data["documents"]:
+            document_family_import_ids.append(entity["family_import_id"])
+
+    event_family_import_ids = []
```

Comment:
Can we make a helper function for this too? Some repeated code

Reply:
I'd prefer to keep it simple for now and refactor once this function is finished as part of [PDCT-1402](https://linear.app/climate-policy-radar/issue/PDCT-1402/validate-collection-exists-before-creating-family) as it might change significantly at that point.